Samples/AppLifecycle/RestartRegistration/cs-winui-packaged/SampleConfiguration.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace cs_winui_packaged
+{
+    public static class Kernel32
+    {
+        public delegate int RecoveryDelegate(IntPtr parameterData);
+
+        [DllImport("kernel32.dll")]
```

Comment:
Shouldn't you have been able to do all these with CSWin32 and NativeMethods.txt?

Reply:
I was actually not familiar with CSwin32 p/invoke generator. Will update to use that.

Reply:
updated